### DISCUSSION TITLE
Add the possibility to compute the NormalizeIntensity inverse

### DISCUSSION THREAD

**Discussion by sh3rlock14** [Project-MONAI/MONAI#5645] [Ideas]
When applying the NormalizeIntensity transform (w/o passing subtrahend and divisor), the computed `_sub `and `_div ` should be added to the meta information. This would be helpful in getting back the original intensity values: not only this could be useful for recon. problems, but also for logging.
I tried to expand NormalizeIntensity and NormalizeIntensityd funcs by updating the metadata, but unsuccessfully.

**Comment by wyli** [comment]
sure, let me convert this into a feature request (https://github.com/Project-MONAI/MONAI/issues/5647)
